props/blaster_BC_buttons.h

```diff
@@ -0,0 +1,807 @@
+                            Self-Destructed blaster needs to be powered on manually to use again.
+Quote Player              - Hold MODE and click FIRE. (requires quote.wavs in font)
+Reload                    - Click MODE.
+Fire                      - Click FIRE. (Hold to Auto / Rapid Fire when AUTO mode selected)
+Clip In                   - Clip Detect pad Latched On.
+Clip out                  - Clip Detect pad Latched Off.
+Unjam                     - Bang the blaster or Reload.
+
+- If there's a dedicated POWER button,
+Power On / Off            - Click POW. - TODO: this with dual_prop is....weird
+- If there's a dedicated Relod button, 
+Reload                    - Click RELOAD.
+
+-TODO: PLI_ON / OFF, CLIP IN etc....
+
+Wavs to use for talking Mode (else Talkie voice):
+- If these are not present, mode.wav will be used for all modes.
+- If no mode.wav either, then Talkie voice speaks selected mode.
+  mdstun.wav
+  mdkill.wav
+  mdauto.wav
+Additionally:
+  quote.wav      // for playing quotes
+  in.wav         // for blaster power off
+  out.wav        // for blaster power on
+  destruct.wav   // for self destruct overload
+  boom.wav       // for self destruct explosion
+  clipins.wv
+  clipouts.wav
+  reloads.wav    // these last 3 are used to match the more "plasmatic" stun sound.
+These are likely best kept in a 'common' folder in your Font Search Path:
+  battery.wav    // for EFFECT_BATTERY_LEVEL
+  vmbegin.wav    // for Begin Volume Menu
+  vmend.wav      // for End Volume Menu
+  volup.wav      // for increse volume
+  voldown.wav    // for decrease volume
+  volmin.wav     // for minimum volume reached
+  volmax.wav     // for maximum volume reached
+
+*/
+
+#ifndef PROPS_BLASTER_BC_H
+#define PROPS_BLASTER_BC_H
+
+#include "prop_base.h"
+#include "../sound/sound_library.h"
+
+#define PROP_TYPE BlasterBC
+#define PROP_HAS_BULLET_COUNT
+
+EFFECT(clipin);
+EFFECT(clipins);    // s for stun mode
+EFFECT(clipouts);   // s for stun mode
+EFFECT(clipout);
+EFFECT(empty);
+EFFECT(full);
+EFFECT(jam);
+EFFECT(mode);
+EFFECT(plioff);
+EFFECT(plion);
+EFFECT(range);
+EFFECT(reload);
+EFFECT(reloads);    // s for stun mode
+EFFECT(stun);
+EFFECT(unjam);
+EFFECT(mdstun);
+EFFECT(mdkill);
+EFFECT(mdauto);
+EFFECT(destruct);
+// For mode sounds, specific "mdstun", "mdkill", and "mdauto" may be used.
+// If just a single "mode" sound for all switches exists, that will be used.
+// If no mode sounds exist in the font, a talkie version will speak the mode on switching.
+#ifndef PROPS_DUAL_PROP_H
+EFFECT(battery);    // for EFFECT_BATTERY_LEVEL
+EFFECT(vmbegin);    // for Begin Volume Menu
+EFFECT(vmend);      // for End Volume Menu
+EFFECT(volup);      // for increse volume
+EFFECT(voldown);    // for decrease volume
+EFFECT(volmin);     // for minimum volume reached
+EFFECT(volmax);     // for maximum volume reached
+EFFECT(quote);      // for playing quotes
+#endif
+
+class BlasterBC : public virtual PropBase, StateMachine {
```

Comment:
1. don't need state machine (see other review)
2. Can we inherit from blaster instead of PropBase to reduce duplication?
3. don't use "virtual" directly, use the define.
4.

Reply:
1. State Machine removed.
2. I'm not sure why or how to do that.
3. I tried to follow this clue several ways with no luck.